### PR TITLE
Update give-mailbox-permissions-to-another-user.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note stating that permissions can be set only within the tenant and not for out of tenant users

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2200

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Once you've set up the permissions, it can take up to 60 minutes for the changes
  **How to use it once permissions are set up:**
   
 There are a few different ways you can access a mailbox once you've been given access. For help on this, refer to this article: [Access another person's mailbox](https://support.microsoft.com/office/A909AD30-E413-40B5-A487-0EA70B763081)
+
+> [!NOTE]
+> The Permissions can be setup only within the current organization tenant. It is not possible to setup mailbox permissions with out of tenant Users.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md]:
```suggestion
> The permissions can be set up only within the current organization tenant. It is not possible to set up mailbox permissions with out of tenant users.
```

File: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md [hunk 2]
```diff
@@ -51,6 +51,9 @@ Once you've set up the permissions, it can take up to 60 minutes for the changes
  **How to use it once permissions are set up:**
   
 There are a few different ways you can access a mailbox once you've been given access. For help on this, refer to this article: [Access another person's mailbox](https://support.microsoft.com/office/A909AD30-E413-40B5-A487-0EA70B763081)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md]:
```suggestion
There are a few different ways you can access a mailbox once you've been given access. For help on this, refer to this article: [Access another person's mailbox](https://support.microsoft.com/office/A909AD30-E413-40B5-A487-0EA70B763081).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KwekuA Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**